src/compressor/CMakeLists.txt

```diff
@@ -23,8 +23,17 @@ if(HAVE_BROTLI)
 endif()
 
 add_library(compressor STATIC ${compressor_srcs})
-if(HAVE_QATZIP)
-  target_link_libraries(compressor PRIVATE ${QATZIP_LIBRARIES})
+if(HAVE_QATZIP AND HAVE_QATDRV)
+  target_link_libraries(compressor_objs PRIVATE
```

Comment:
@tchaikov the problem maybe found: the object library `compressor_objs` missed its dependencies. Here, we use `target_link_libraries()` to add the dependencies. And the line` include_directories("${QatDrv_INCLUDE_DIRS}")` has been deleted.